docs/resources/rds_database.md

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Relational Database Service (RDS)"
+---
+
+# huaweicloud_rds_database
+
+Manages RDS Mysql database resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "instance_id" {}
+
+resource "huaweicloud_rds_database" "test" {
+  instance_id   = huaweicloud_rds_instance.test.id
+  name          = var.instance_id
```

Comment:
ditto

Reply:
this should be used as intance_id, fixed